src/main/java/pl/project13/maven/git/GitCommitIdMojo.java

```diff
@@ -631,21 +633,26 @@ static Properties readJsonProperties(@NotNull File jsonFile) throws CannotReadFi
       Closeable closeable = null;
 
       try {
-        final FileInputStream fis = new FileInputStream(jsonFile);
-        closeable = fis;
-
-        final InputStreamReader reader = new InputStreamReader(fis, Charsets.UTF_8);
-        closeable = reader;
-
-        final ObjectMapper mapper = new ObjectMapper();
-        final TypeReference<HashMap<String,Object>> mapTypeRef =
-          new TypeReference<HashMap<String,Object>>() {};
-
-        propertiesMap = mapper.readValue(reader, mapTypeRef);
+        boolean threw = true;
+        try {
```

Comment:
It cannot, as [Closeable.close()](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/io/Closeables.html#close(java.io.Closeable,%20boolean)) declares `throws IOException` and this exception needs to be repacked into unchecked `CannotReadFileException`.